src/SignalR/clients/ts/FunctionalTests/ts/Common.ts

```diff
@@ -1,8 +1,9 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-import { HttpTransportType, IHubProtocol, JsonHubProtocol } from "@microsoft/signalr";
+import { DefaultHttpClient, FetchHttpClient, HttpClient, HttpTransportType, IHubProtocol, JsonHubProtocol, XhrHttpClient } from "@microsoft/signalr";
```

Comment:
Don't export these types, instead import them from "@microsoft/signalr/dist/esm/..." like we do [here](https://github.com/aspnet/AspNetCore/pull/12599/files#diff-06d677716b385b2fb919584813238af0R11).